benchmarks/benchmark.py

```diff
@@ -0,0 +1,575 @@
+#!/usr/bin/python
+from __future__ import print_function
+from argparse import ArgumentParser, RawTextHelpFormatter
```

Comment:
```suggestion
from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter
```

I think this is a bit better (it shows the defaults too)

Reply:
fixed